x-pack/plugins/observability/public/components/app/section/alerts/index.tsx

```diff
@@ -31,105 +32,118 @@ const allTypes = {
-      title={i18n.translate('xpack.observability.overview.alerts.title', {
-        defaultMessage: 'Alerts',
-      })}
-      appLink={{
-        href: '/app/management/insightsAndAlerting/triggersActions/alerts',
-        label: i18n.translate('xpack.observability.overview.alert.appLink', {
-          defaultMessage: 'Manage alerts',
-        }),
-      }}
-      hasError={false}
-    >
-      <EuiFlexGroup direction="column" gutterSize="none">
-        <EuiFlexItem grow={false}>
-          <EuiFlexGroup justifyContent="flexEnd">
-            <EuiFlexItem grow={false}>
-              <EuiSelect
-                compressed
-                id="filterAlerts"
-                options={[allTypes, ...filterOptions]}
-                value={filter}
-                onChange={(e) => setFilter(e.target.value)}
-                prepend={i18n.translate('xpack.observability.overview.alert.view', {
-                  defaultMessage: 'View',
-                })}
-              />
-            </EuiFlexItem>
-          </EuiFlexGroup>
-        </EuiFlexItem>
-        <EuiHorizontalRule margin="xs" />
-        <EuiFlexItem>
-          {alerts
-            .filter((alert) => filter === ALL_TYPES || alert.consumer === filter)
-            .map((alert, index) => {
-              const isLastElement = index === alerts.length - 1;
-              return (
-                <EuiFlexGroup direction="column" gutterSize="s" key={alert.id}>
-                  <EuiSpacer size="s" />
-                  <EuiFlexItem>
-                    <EuiLink
-                      href={core.http.basePath.prepend(
-                        `/app/management/insightsAndAlerting/triggersActions/alert/${alert.id}`
-                      )}
-                    >
-                      <EuiText size="s">{alert.name}</EuiText>
-                    </EuiLink>
-                  </EuiFlexItem>
-                  <EuiFlexItem grow={false}>
-                    <EuiFlexGroup gutterSize="xs">
-                      <EuiFlexItem grow={false}>
-                        <EuiBadge color="hollow">{alert.alertTypeId}</EuiBadge>
-                      </EuiFlexItem>
-                      {alert.tags.map((tag, idx) => {
-                        return (
-                          <EuiFlexItem key={idx} grow={false}>
-                            <EuiBadge color="default">{tag}</EuiBadge>
-                          </EuiFlexItem>
-                        );
-                      })}
-                    </EuiFlexGroup>
-                  </EuiFlexItem>
-                  <EuiFlexItem>
-                    <EuiFlexGroup gutterSize="s">
-                      <EuiFlexItem grow={false}>
-                        <EuiText color="subdued" size="xs">
-                          Updated {moment.duration(moment().diff(alert.updatedAt)).humanize()} ago
-                        </EuiText>
-                      </EuiFlexItem>
-                      {alert.muteAll && (
+    <EuiFlexItem grow={3}>
```

Comment:
I wouldn't add this here, I think the AlertsSection component should only return the alert section, it will depend on which component called it to defined if it would be wrapped in a `EuiFlexGroup`.

```suggestion
    
```

Reply:
ok, I got why you did it, it's because the alerts are now being fetched in the component itself...